coq/dune

```diff
@@ -3,4 +3,6 @@
  (public_name coq-lsp.coq)
  ; Unfortunate we have to link the STM due to the LTAC plugin
  ; depending on it, we should fix this upstream
+ (inline_tests)
+ (preprocess (pps ppx_inline_test))
```

Comment:
This adds the test library as a dependency to the executable, which is probably not what you want. I think it would be better to have a separate test library like before.

Reply:
Does it? I think dune handles this properly.